ups/eupspkg.cfg.sh

```diff
@@ -67,4 +67,14 @@ install()
 	default_install
 
 	cp -r include "$PREFIX/"
+	if [[ $OSTYPE == darwin* ]]; then
+		curdir=`pwd`
+		cd $PREFIX/lib
+		galsim_name=`ls libgalsim.*.dylib`
+		rm libgalsim.dylib
+		ln -s $galsim_name libgalsim.dylib
+		install_name_tool -id $galsim_name $galsim_name
+		install_name_tool -change $PREFIX/lib/$galsim_name @loader_path/../../$galsim_name python/galsim/_galsim.so
```

Comment:
Why is this necessary? it should be looking for the symlink you just created. This seems to be making the binary point right to the actual library and not the symlink? Maybe I don't understand the install_name_tool well enough. Can you clarify why you need both this and creating a symlink?

Reply:
I thought the symlink was a convenience for packages outside of galsim.  As far as I can tell, `_galsim.so` always pointed to the versioned library not the symlink.  We needed to specify where the library is relative to `_galsim.so` because it wasn't being picked up in the `@rpath`.  I have to admit that I don't fully understand the Apple linking model, so it's certainly possible that there are other ways to do it, but in my testing this is the first incantation that worked for me.

Reply:
I don't fully understand apples linking model either, but I am happy if you are satisfied it is needed and it works.